migrate.py

```diff
@@ -79,6 +113,15 @@ def fail(msg: str) -> NoReturn:
     conn = connect(db_path, isolation_level=None)
     try:
         cursor = conn.cursor()
+
+        def execute_sql(
+            cursor: sqlite3.Cursor, query: str, params: Optional[Tuple[str, ...]] = None
```

Comment:
Since it looks like only the one with `params` is called, would it be easier to just use `cusror.execute` everywhere? (would save some type annotation pain)